gcore/src/exec.rs

```diff
@@ -336,3 +344,26 @@ pub fn origin() -> ActorId {
 
     origin
 }
+
+/// Get the random seed.
```

Comment:
Please, provide extra docs about how this get created and for which purposes it may be used: e.g. never use for gambling